rs/types/types/src/consensus/idkg/schnorr.rs

```diff
@@ -258,6 +258,7 @@ impl ThresholdSchnorrSigInputsRef {
         ThresholdSchnorrSigInputs::new(
             &self.derivation_path,
             &self.message,
+            None, // TODO &self.taproot_tree_root
```

Comment:
Is it intentional that this is still a TODO? If so, how about adding a comment in the PR description.

Reply:
It's a TODO but for a future PR where the aux input is handled by the management canister and send down to crypto via consensus. I'll add a reference to the relevant ticket (CRP-2630) here